src/tools/perf/perftest.c

```diff
@@ -282,6 +286,7 @@ static void print_header(struct perftest_context *ctx)
             printf("| API:          %-60s               |\n", test_api_str);
             printf("| Test:         %-60s               |\n", test->desc);
             printf("| Data layout:  %-60s               |\n", test_data_str);
+            printf("| Memory types: %-60s               |\n", test_mem_str);
```

Comment:
why not print `ucs_memory_type_names[ctx->params.send_mem_type]` and `ucs_memory_type_names[ctx->params.recv_mem_type]` directly here? So, `test_mem_str` can be removed in this case
UCT perftest also supports different memtypes

Reply:
We want to print both the send and recv memory types. Since these two strings can of different lengths, making sure that the print is properly aligned to 60 characters becomes complicated.

Reply:
maybe better separate it to:
```
printf("| Send memory type    : %-60s               |\n", ucs_memory_type_names[ctx->params.send_mem_type]);
printf("| Receive memory type: %-60s               |\n", ucs_memory_type_names[ctx->params.recv_mem_type]);
```

Reply:
@souravzzz wdyt about separating them?

Reply:
Wanted to keep the print similar to that of data layout, which also prints the src and the dst types. But I agree this makes is more clear.